src/renderer/vt/VtSequences.cpp

```diff
@@ -234,7 +232,10 @@ using namespace Microsoft::Console::Render;
                         (WI_IsFlagSet(wAttr, FOREGROUND_GREEN) ? 2 : 0) +
                         (WI_IsFlagSet(wAttr, FOREGROUND_BLUE) ? 4 : 0);
 
-    return _WriteFormattedString(&fmt, vtIndex);
+    // An example string with max length would be "\x1b[100m", which has length = 6.

```

Comment:
Ideally this is expected. I wonder if we should have more spaces in buffer, just in case the arguments are not as small as we expected.